.github/workflows/nightly.yml

```diff
@@ -36,7 +36,7 @@ jobs:
           path: screencap.png
 
   unit_test_and_release:
-    runs-on: ubuntu-latest
+    runs-on: ubuntu-18.04
```

Comment:
why? What is wrong with ubuntu-latest?

Reply:
https://github.com/actions/virtual-environments/issues/1816

Reply:
Yes, and what is the problem with ubuntu 20.20?

Reply:
As mentioned in #android at slack, we haven't yet configured project for it